File: csharp/module1/solutions/solution4.cs
public int GetPlayerExperience(int numOfLevel1CreaturesKilled, int numOfLevel2CreaturesKilled) {
    const int tutorialPoints = 1000;
    const int pointsForKillingLevel1Creature = 500;
    const int pointsForKillingLevel2Creature = 1000;

    int pointsForLevel1CreaturesKilled = numOfLevel1CreaturesKilled * pointsForKillingLevel1Creature;
    int pointsForLevel2CreaturesKilled = numOfLevel2CreaturesKilled * pointsForKillingLevel2Creature;

    return tutorialPoints + pointsForLevel1CreaturesKilled + pointsForLevel2CreaturesKilled;
}

File: javascript/module1/solutions/solution4.js
function getPlayerExprience(
    numOfLevel1CreaturesKilled,
    numOfLevel2CreaturesKilled
  ) {
  const tutorialPoints = 1000;
  const pointsForKillingLevel1Creature = 500;
  const pointsForKillingLevel2Creature = 1000;
  
  const pointsForLevel1CreaturesKilled =
   numOfLevel1CreaturesKilled * pointsForKillingLevel1Creature;
  const pointsForLevel2CreaturesKilled =
   numOfLevel2CreaturesKilled * pointsForKillingLevel2Creature;
  
    // 1000 points for finishing tutorial
  return tutorialPoints + pointsForLevel1CreaturesKilled + pointsForLevel2CreaturesKilled;
}
